dev/com.ibm.ws.webcontainer.security/src/com/ibm/ws/webcontainer/security/WebAppSecurityCollaboratorImpl.java

```diff
@@ -567,8 +576,10 @@ public Object preInvoke(HttpServletRequest req, HttpServletResponse resp, String
                 performSecurityChecks(req, resp, receivedSubject, webSecurityContext);
             }
 
-            extraAuditData.put("HTTP_SERVLET_REQUEST", req);
```

Comment:
I was wondering why the extra audit data was deleted as this doesn't seem to have anything to do with the sync to thread changes, but then again I don't know much about these parts so maybe it was necessary =)